[debug] [2025-06-15T18:45:46.724Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-06-15T18:45:55.418Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-06-15T18:45:55.423Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-15T18:45:55.424Z] > authorizing via signed-in user (ancientastronautsunearthed@gmail.com)
[debug] [2025-06-15T18:45:55.435Z] Checked if tokens are valid: true, expires at: 1750016675979
[debug] [2025-06-15T18:45:55.435Z] Checked if tokens are valid: true, expires at: 1750016675979
[debug] [2025-06-15T18:45:55.436Z] [Firebase Plugin] User found:  ancientastronautsunearthed@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-06-15T18:45:55.439Z] Checked if tokens are valid: true, expires at: 1750016675979
[debug] [2025-06-15T18:45:55.442Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-06-15T18:45:55.474Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-06-15T18:45:56.222Z] Object "/apphosting" in "firebase.json" is missing required property: {"missingProperty":"backendId"}
[debug] [2025-06-15T18:45:56.222Z] Field "/apphosting" in "firebase.json" is possibly invalid: must be array
[debug] [2025-06-15T18:45:56.223Z] Field "/apphosting" in "firebase.json" is possibly invalid: must match a schema in anyOf
[debug] [2025-06-15T18:45:56.711Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-06-15T18:45:56.711Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Stopping Data Connect toolkit
[debug] [2025-06-15T21:21:45.614Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-06-15T21:21:50.279Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-06-15T21:21:50.282Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-15T21:21:50.283Z] > authorizing via signed-in user (ancientastronautsunearthed@gmail.com)
[debug] [2025-06-15T21:21:50.288Z] Checked if tokens are valid: true, expires at: 1750023848630
[debug] [2025-06-15T21:21:50.289Z] Checked if tokens are valid: true, expires at: 1750023848630
[debug] [2025-06-15T21:21:50.289Z] [Firebase Plugin] User found:  ancientastronautsunearthed@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-06-15T21:21:50.292Z] Checked if tokens are valid: true, expires at: 1750023848630
[debug] [2025-06-15T21:21:50.294Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-06-15T21:21:50.308Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-06-15T21:21:52.795Z] Object "/hosting/frameworksBackend" in "firebase.json" has unknown property: {"additionalProperty":"id"}
[debug] [2025-06-15T21:21:52.797Z] Field "/hosting" in "firebase.json" is possibly invalid: must be array
[debug] [2025-06-15T21:21:52.798Z] Field "/hosting" in "firebase.json" is possibly invalid: must match a schema in anyOf
[debug] [2025-06-15T21:21:53.244Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-06-15T21:21:53.245Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-06-16T03:23:33.332Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-06-16T03:23:39.217Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-06-16T03:23:39.231Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-16T03:23:39.232Z] > authorizing via signed-in user (ancientastronautsunearthed@gmail.com)
[debug] [2025-06-16T03:23:39.239Z] Checked if tokens are valid: false, expires at: 1750026674143
[debug] [2025-06-16T03:23:39.240Z] Checked if tokens are valid: false, expires at: 1750026674143
[debug] [2025-06-16T03:23:39.241Z] > refreshing access token with scopes: []
[debug] [2025-06-16T03:23:39.245Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-06-16T03:23:39.245Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-06-16T03:23:39.268Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-06-16T03:23:40.081Z] Field "/functions/runtime" in "firebase.json" is possibly invalid: must be equal to one of the allowed values
[debug] [2025-06-16T03:23:40.082Z] Field "/functions" in "firebase.json" is possibly invalid: must be array
[debug] [2025-06-16T03:23:40.082Z] Field "/functions" in "firebase.json" is possibly invalid: must match a schema in anyOf
[debug] [2025-06-16T03:23:40.103Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-06-16T03:23:40.104Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-06-16T03:23:40.130Z] [Firebase Plugin] User found:  ancientastronautsunearthed@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-06-16T03:23:40.143Z] Checked if tokens are valid: true, expires at: 1750047819104
[debug] [2025-06-16T03:23:40.145Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-06-16T03:23:40.552Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-06-16T03:23:40.553Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Stopping Data Connect toolkit
[debug] [2025-06-17T05:48:08.669Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-06-17T05:48:14.660Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-06-17T05:48:14.675Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-17T05:48:14.676Z] > authorizing via signed-in user (ancientastronautsunearthed@gmail.com)
[debug] [2025-06-17T05:48:14.683Z] Checked if tokens are valid: false, expires at: 1750129651524
[debug] [2025-06-17T05:48:14.684Z] Checked if tokens are valid: false, expires at: 1750129651524
[debug] [2025-06-17T05:48:14.684Z] > refreshing access token with scopes: []
[debug] [2025-06-17T05:48:14.686Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-06-17T05:48:14.687Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-06-17T05:48:14.706Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-06-17T05:48:15.167Z] Field "/functions/runtime" in "firebase.json" is possibly invalid: must be equal to one of the allowed values
[debug] [2025-06-17T05:48:15.168Z] Field "/functions" in "firebase.json" is possibly invalid: must be array
[debug] [2025-06-17T05:48:15.168Z] Field "/functions" in "firebase.json" is possibly invalid: must match a schema in anyOf
[debug] [2025-06-17T05:48:15.255Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-06-17T05:48:15.256Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-06-17T05:48:15.287Z] [Firebase Plugin] User found:  ancientastronautsunearthed@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-06-17T05:48:15.290Z] Checked if tokens are valid: true, expires at: 1750142894256
[debug] [2025-06-17T05:48:15.292Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-06-17T05:48:15.877Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-06-17T05:48:15.880Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-06-17T06:27:32.543Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-06-17T06:27:38.483Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-06-17T06:27:38.488Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-17T06:27:38.489Z] > authorizing via signed-in user (ancientastronautsunearthed@gmail.com)
[debug] [2025-06-17T06:27:38.498Z] Checked if tokens are valid: true, expires at: 1750142894256
[debug] [2025-06-17T06:27:38.499Z] Checked if tokens are valid: true, expires at: 1750142894256
[debug] [2025-06-17T06:27:38.500Z] [Firebase Plugin] User found:  ancientastronautsunearthed@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-06-17T06:27:38.505Z] Checked if tokens are valid: true, expires at: 1750142894256
[debug] [2025-06-17T06:27:38.508Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-06-17T06:27:38.531Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-06-17T06:27:39.366Z] Object "/apphosting" in "firebase.json" is missing required property: {"missingProperty":"backendId"}
[debug] [2025-06-17T06:27:39.369Z] Field "/apphosting" in "firebase.json" is possibly invalid: must be array
[debug] [2025-06-17T06:27:39.370Z] Field "/apphosting" in "firebase.json" is possibly invalid: must match a schema in anyOf
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.6.3) on port 50001
[debug] [2025-06-17T06:27:39.975Z] 'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0616 23:27:39.956415   15916 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0616 23:27:39.956415   15916 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:27:39.976Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-06-17T06:27:39.977Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-06-17T06:27:39.977Z] Starting Data Connect Emulator with command {"binary":"C:\\Users\\cmill\\.cache\\firebase\\emulators\\dataconnect-emulator-2.6.3.exe","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"C:\\\\Users\\\\cmill\\\\.cache\\\\firebase\\\\emulators\\\\dataconnect-emulator-2.6.3.exe\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to \u001b[1mdataconnect-debug.log\u001b[22m"}}
[debug] [2025-06-17T06:27:40.239Z] E0616 23:27:40.239116    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:27:40.239116    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:27:40.250Z] E0616 23:27:40.247408    8088 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:27:40.247408    8088 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T06:27:40.292Z] I0616 23:27:40.292532    8088 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0616 23:27:40.292532    8088 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-06-17T06:27:40.365Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-06-17T06:27:40.366Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[error] [Firebase Plugin] error:  {}
[debug] [2025-06-17T06:27:50.249Z] E0616 23:27:50.259308    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:27:50.259308    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:28:00.240Z] E0616 23:28:00.251387    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:28:00.251387    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:28:18.113Z] E0616 23:28:10.269268    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:28:10.269268    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:28:20.240Z] E0616 23:28:20.251136    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:28:20.251136    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:28:30.253Z] E0616 23:28:30.264355    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:28:30.264355    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:28:40.248Z] E0616 23:28:40.259791    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:28:40.259791    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:28:50.407Z] E0616 23:28:50.406929    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:28:50.406929    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:29:00.261Z] E0616 23:29:00.261589    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:29:00.261589    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:29:10.258Z] E0616 23:29:10.257624    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:29:10.257624    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:29:20.259Z] E0616 23:29:20.258234    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:29:20.258234    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:29:30.261Z] E0616 23:29:30.260593    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:29:30.260593    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:29:40.264Z] E0616 23:29:40.262939    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:29:40.262939    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:29:40.267Z] E0616 23:29:40.263685    8088 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:29:40.263685    8088 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T06:29:50.263Z] E0616 23:29:50.262652    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:29:50.262652    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:30:00.251Z] E0616 23:30:00.251942    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:30:00.251942    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:30:10.267Z] E0616 23:30:10.267045    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:30:10.267045    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:30:20.260Z] E0616 23:30:20.259947    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:30:20.259947    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:30:30.264Z] E0616 23:30:30.264276    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:30:30.264276    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:30:40.259Z] E0616 23:30:40.259552    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:30:40.259552    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:30:50.266Z] E0616 23:30:50.265688    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:30:50.265688    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:31:00.598Z] E0616 23:31:00.327154    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:31:00.327154    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:31:10.293Z] E0616 23:31:10.265044    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:31:10.265044    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:31:20.356Z] E0616 23:31:20.355429    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:31:20.355429    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:31:30.253Z] E0616 23:31:30.253492    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:31:30.253492    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:31:40.257Z] E0616 23:31:40.256943    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:31:40.256943    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:31:40.337Z] E0616 23:31:40.337579    8088 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:31:40.337579    8088 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T06:31:50.253Z] E0616 23:31:50.253125    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:31:50.253125    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:32:00.255Z] E0616 23:32:00.255095    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:32:00.255095    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:32:10.258Z] E0616 23:32:10.258292    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:32:10.258292    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:32:20.265Z] E0616 23:32:20.264934    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:32:20.264934    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:32:30.254Z] E0616 23:32:30.253720    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:32:30.253720    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:32:40.260Z] E0616 23:32:40.260540    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:32:40.260540    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:32:50.255Z] E0616 23:32:50.254958    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:32:50.254958    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:33:00.253Z] E0616 23:33:00.252936    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:33:00.252936    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:33:10.268Z] E0616 23:33:10.268484    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:33:10.268484    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:33:20.257Z] E0616 23:33:20.256986    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:33:20.256986    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:33:40.081Z] E0616 23:33:30.254851    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:33:30.254851    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:33:40.351Z] E0616 23:33:40.332290    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
E0616 23:33:40.332799    8088 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:33:40.332290    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\nE0616 23:33:40.332799    8088 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T06:33:50.259Z] E0616 23:33:50.259098    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:33:50.259098    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:34:00.286Z] E0616 23:34:00.286296    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:34:00.286296    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:34:10.256Z] E0616 23:34:10.256551    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:34:10.256551    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:34:20.254Z] E0616 23:34:20.254616    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:34:20.254616    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:34:30.253Z] E0616 23:34:30.253060    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:34:30.253060    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:34:40.255Z] E0616 23:34:40.255269    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:34:40.255269    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:34:50.266Z] E0616 23:34:50.265789    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:34:50.265789    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:35:00.309Z] E0616 23:35:00.308978    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:35:00.308978    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:35:10.261Z] E0616 23:35:10.261789    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:35:10.261789    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:35:20.263Z] E0616 23:35:20.264004    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:35:20.264004    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:35:30.336Z] E0616 23:35:30.323748    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:35:30.323748    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:35:40.257Z] E0616 23:35:40.257549    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:35:40.257549    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:35:40.258Z] E0616 23:35:40.258056    8088 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:35:40.258056    8088 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T06:35:50.389Z] E0616 23:35:50.383191    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:35:50.383191    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:36:00.256Z] E0616 23:36:00.264604    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:36:00.264604    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:36:10.253Z] E0616 23:36:10.262373    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:36:10.262373    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:36:20.256Z] E0616 23:36:20.259939    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:36:20.259939    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:36:30.245Z] E0616 23:36:30.254346    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:36:30.254346    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:36:40.253Z] E0616 23:36:40.262379    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:36:40.262379    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:36:50.250Z] E0616 23:36:50.259908    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:36:50.259908    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:37:00.267Z] E0616 23:37:00.266331    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:37:00.266331    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:37:10.259Z] E0616 23:37:10.258579    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:37:10.258579    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:37:20.257Z] E0616 23:37:20.257744    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:37:20.257744    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:37:30.264Z] E0616 23:37:30.263602    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:37:30.263602    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:37:40.277Z] E0616 23:37:40.276459    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:37:40.276459    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:37:40.279Z] E0616 23:37:40.276459    8088 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:37:40.276459    8088 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T06:37:50.257Z] E0616 23:37:50.256740    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:37:50.256740    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:38:00.258Z] E0616 23:38:00.257881    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:38:00.257881    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:38:10.251Z] E0616 23:38:10.251605    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:38:10.251605    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:38:20.257Z] E0616 23:38:20.257494    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:38:20.257494    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:38:30.257Z] E0616 23:38:30.256613    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:38:30.256613    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:38:40.254Z] E0616 23:38:40.254020    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:38:40.254020    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:38:50.265Z] E0616 23:38:50.265482    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:38:50.265482    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:39:00.252Z] E0616 23:39:00.252438    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:39:00.252438    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:39:10.265Z] E0616 23:39:10.265792    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:39:10.265792    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:39:20.264Z] E0616 23:39:20.265121    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:39:20.265121    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:39:30.259Z] E0616 23:39:30.259314    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:39:30.259314    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:39:40.573Z] E0616 23:39:40.441464    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
E0616 23:39:40.441464    8088 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:39:40.441464    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\nE0616 23:39:40.441464    8088 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T06:39:50.264Z] E0616 23:39:50.264099    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:39:50.264099    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:40:00.256Z] E0616 23:40:00.257288    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:40:00.257288    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:40:10.264Z] E0616 23:40:10.263870    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:40:10.263870    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:40:20.258Z] E0616 23:40:20.258872    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:40:20.258872    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:40:30.263Z] E0616 23:40:30.262755    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:40:30.262755    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:40:40.260Z] E0616 23:40:40.259829    8088 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:40:40.259829    8088 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] !  Data Connect Emulator has exited upon receiving signal: SIGKILL 
[debug] [2025-06-17T06:57:29.348Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-06-17T06:57:34.331Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-06-17T06:57:34.335Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-17T06:57:34.336Z] > authorizing via signed-in user (ancientastronautsunearthed@gmail.com)
[debug] [2025-06-17T06:57:34.342Z] Checked if tokens are valid: true, expires at: 1750146027352
[debug] [2025-06-17T06:57:34.343Z] Checked if tokens are valid: true, expires at: 1750146027352
[debug] [2025-06-17T06:57:34.343Z] [Firebase Plugin] User found:  ancientastronautsunearthed@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-06-17T06:57:34.345Z] Checked if tokens are valid: true, expires at: 1750146027352
[debug] [2025-06-17T06:57:34.347Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-06-17T06:57:34.369Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-06-17T06:57:35.002Z] Object "/apphosting" in "firebase.json" is missing required property: {"missingProperty":"backendId"}
[debug] [2025-06-17T06:57:35.004Z] Field "/apphosting" in "firebase.json" is possibly invalid: must be array
[debug] [2025-06-17T06:57:35.005Z] Field "/apphosting" in "firebase.json" is possibly invalid: must match a schema in anyOf
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.6.3) on port 50001
[debug] [2025-06-17T06:57:35.216Z] 'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0616 23:57:35.212218   16748 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0616 23:57:35.212218   16748 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:57:35.218Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-06-17T06:57:35.218Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-06-17T06:57:35.220Z] Starting Data Connect Emulator with command {"binary":"C:\\Users\\cmill\\.cache\\firebase\\emulators\\dataconnect-emulator-2.6.3.exe","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"C:\\\\Users\\\\cmill\\\\.cache\\\\firebase\\\\emulators\\\\dataconnect-emulator-2.6.3.exe\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to \u001b[1mdataconnect-debug.log\u001b[22m"}}
[debug] [2025-06-17T06:57:35.330Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-06-17T06:57:35.330Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-06-17T06:57:35.355Z] E0616 23:57:35.355512   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:57:35.355512   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:57:35.357Z] E0616 23:57:35.357728   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:57:35.357728   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T06:57:35.379Z] I0616 23:57:35.379644   10992 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0616 23:57:35.379644   10992 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[error] [Firebase Plugin] error:  {}
[debug] [2025-06-17T06:57:45.367Z] E0616 23:57:45.368079   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:57:45.368079   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:57:55.358Z] E0616 23:57:55.357871   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:57:55.357871   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:58:05.366Z] E0616 23:58:05.367436   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:58:05.367436   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:58:15.371Z] E0616 23:58:15.371825   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:58:15.371825   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:58:25.364Z] E0616 23:58:25.364779   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:58:25.364779   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:58:35.363Z] E0616 23:58:35.363743   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:58:35.363743   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:58:45.364Z] E0616 23:58:45.364522   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:58:45.364522   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:58:55.374Z] E0616 23:58:55.374137   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:58:55.374137   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:59:05.373Z] E0616 23:59:05.372781   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:59:05.372781   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:59:15.521Z] E0616 23:59:15.368266   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:59:15.368266   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:59:25.379Z] E0616 23:59:25.380051   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:59:25.380051   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:59:35.370Z] E0616 23:59:35.370652   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:59:35.370652   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:59:35.371Z] E0616 23:59:35.370652   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:59:35.370652   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T06:59:45.406Z] E0616 23:59:45.400351   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:59:45.400351   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T06:59:55.369Z] E0616 23:59:55.369686   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0616 23:59:55.369686   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:00:05.371Z] E0617 00:00:05.371735   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:00:05.371735   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:00:15.364Z] E0617 00:00:15.364773   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:00:15.364773   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:00:25.362Z] E0617 00:00:25.362262   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:00:25.362262   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:00:35.374Z] E0617 00:00:35.374282   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:00:35.374282   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:00:45.360Z] E0617 00:00:45.359941   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:00:45.359941   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:00:55.364Z] E0617 00:00:55.363957   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:00:55.363957   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:01:05.370Z] E0617 00:01:05.370181   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:01:05.370181   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:01:15.364Z] E0617 00:01:15.364127   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:01:15.364127   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:01:25.372Z] E0617 00:01:25.372725   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:01:25.372725   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:01:35.371Z] E0617 00:01:35.371375   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:01:35.371375   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:01:35.372Z] E0617 00:01:35.371916   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:01:35.371916   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:01:45.368Z] E0617 00:01:45.367589   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:01:45.367589   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:01:55.364Z] E0617 00:01:55.364427   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:01:55.364427   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:02:05.365Z] E0617 00:02:05.365242   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:02:05.365242   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:02:15.360Z] E0617 00:02:15.360711   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:02:15.360711   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:02:25.365Z] E0617 00:02:25.361223   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:02:25.361223   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:02:35.369Z] E0617 00:02:35.373094   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:02:35.373094   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:02:45.365Z] E0617 00:02:45.369348   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:02:45.369348   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:02:55.357Z] E0617 00:02:55.361209   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:02:55.361209   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:03:05.354Z] E0617 00:03:05.358481   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:03:05.358481   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:03:15.355Z] E0617 00:03:15.359902   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:03:15.359902   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:03:25.359Z] E0617 00:03:25.363388   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:03:25.363388   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:03:35.374Z] E0617 00:03:35.364921   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
E0617 00:03:35.364921   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:03:35.364921   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\nE0617 00:03:35.364921   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:03:45.360Z] E0617 00:03:45.360281   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:03:45.360281   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:03:55.362Z] E0617 00:03:55.361906   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:03:55.361906   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:04:05.370Z] E0617 00:04:05.370487   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:04:05.370487   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:04:15.361Z] E0617 00:04:15.361008   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:04:15.361008   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:04:25.358Z] E0617 00:04:25.358952   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:04:25.358952   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:04:35.361Z] E0617 00:04:35.360523   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:04:35.360523   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:04:45.372Z] E0617 00:04:45.372413   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:04:45.372413   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:04:55.386Z] E0617 00:04:55.381159   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:04:55.381159   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:05:05.371Z] E0617 00:05:05.371056   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:05:05.371056   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:05:15.370Z] E0617 00:05:15.370436   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:05:15.370436   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:05:25.371Z] E0617 00:05:25.371588   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:05:25.371588   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:05:35.370Z] E0617 00:05:35.370120   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:05:35.370120   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:05:35.371Z] E0617 00:05:35.370120   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:05:35.370120   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:05:45.369Z] E0617 00:05:45.369752   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:05:45.369752   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:05:55.368Z] E0617 00:05:55.368200   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:05:55.368200   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:06:05.367Z] E0617 00:06:05.367372   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:06:05.367372   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:06:15.372Z] E0617 00:06:15.372554   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:06:15.372554   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:06:25.405Z] E0617 00:06:25.401005   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:06:25.401005   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:06:35.364Z] E0617 00:06:35.364645   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:06:35.364645   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:06:45.361Z] E0617 00:06:45.361188   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:06:45.361188   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:06:55.362Z] E0617 00:06:55.362276   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:06:55.362276   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:07:05.361Z] E0617 00:07:05.360379   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:07:05.360379   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:07:15.363Z] E0617 00:07:15.362715   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:07:15.362715   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:07:25.363Z] E0617 00:07:25.363051   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:07:25.363051   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:07:35.361Z] E0617 00:07:35.361316   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:07:35.361316   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:07:35.362Z] E0617 00:07:35.361316   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:07:35.361316   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:07:45.361Z] E0617 00:07:45.361071   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:07:45.361071   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:07:55.360Z] E0617 00:07:55.359621   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:07:55.359621   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:08:05.357Z] E0617 00:08:05.358429   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:08:05.358429   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:08:15.359Z] E0617 00:08:15.360564   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:08:15.360564   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:08:25.359Z] E0617 00:08:25.360437   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:08:25.360437   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:08:35.367Z] E0617 00:08:35.367977   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:08:35.367977   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:08:45.360Z] E0617 00:08:45.361545   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:08:45.361545   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:08:55.370Z] E0617 00:08:55.372376   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:08:55.372376   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:09:05.370Z] E0617 00:09:05.371981   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:09:05.371981   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:09:15.372Z] E0617 00:09:15.374021   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:09:15.374021   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:09:25.361Z] E0617 00:09:25.363440   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:09:25.363440   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:09:35.361Z] E0617 00:09:35.363694   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:09:35.363694   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:09:35.364Z] E0617 00:09:35.363694   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:09:35.363694   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:09:45.360Z] E0617 00:09:45.362520   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:09:45.362520   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:09:55.366Z] E0617 00:09:55.365719   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:09:55.365719   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:10:05.369Z] E0617 00:10:05.367363   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:10:05.367363   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:10:15.371Z] E0617 00:10:15.371447   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:10:15.371447   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:10:25.371Z] E0617 00:10:25.371045   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:10:25.371045   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:10:35.362Z] E0617 00:10:35.361893   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:10:35.361893   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:10:45.360Z] E0617 00:10:45.360891   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:10:45.360891   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:10:55.361Z] E0617 00:10:55.361179   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:10:55.361179   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:11:05.362Z] E0617 00:11:05.362193   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:11:05.362193   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:11:15.361Z] E0617 00:11:15.360771   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:11:15.360771   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:11:25.359Z] E0617 00:11:25.359051   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:11:25.359051   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:11:35.358Z] E0617 00:11:35.357977   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
E0617 00:11:35.358529   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:11:35.357977   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\nE0617 00:11:35.358529   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:11:45.367Z] E0617 00:11:45.366558   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:11:45.366558   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:11:55.359Z] E0617 00:11:55.358653   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:11:55.358653   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:12:05.360Z] E0617 00:12:05.360345   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:12:05.360345   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:12:15.376Z] E0617 00:12:15.375848   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:12:15.375848   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:12:25.376Z] E0617 00:12:25.374478   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:12:25.374478   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:12:35.368Z] E0617 00:12:35.367619   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:12:35.367619   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:12:45.368Z] E0617 00:12:45.368088   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:12:45.368088   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:12:55.361Z] E0617 00:12:55.361161   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:12:55.361161   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:13:05.368Z] E0617 00:13:05.367802   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:13:05.367802   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:13:15.358Z] E0617 00:13:15.358097   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:13:15.358097   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:13:25.373Z] E0617 00:13:25.373136   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:13:25.373136   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:13:35.371Z] E0617 00:13:35.370944   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
E0617 00:13:35.370944   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:13:35.370944   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\nE0617 00:13:35.370944   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:13:45.364Z] E0617 00:13:45.363823   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:13:45.363823   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:13:55.381Z] E0617 00:13:55.379759   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:13:55.379759   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:14:05.367Z] E0617 00:14:05.366633   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:14:05.366633   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:14:15.369Z] E0617 00:14:15.369143   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:14:15.369143   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:14:25.367Z] E0617 00:14:25.367663   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:14:25.367663   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:14:35.374Z] E0617 00:14:35.374470   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:14:35.374470   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:14:45.362Z] E0617 00:14:45.362959   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:14:45.362959   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:14:55.362Z] E0617 00:14:55.362286   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:14:55.362286   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:15:05.371Z] E0617 00:15:05.371949   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:15:05.371949   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:15:15.361Z] E0617 00:15:15.361157   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:15:15.361157   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:15:25.362Z] E0617 00:15:25.361903   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:15:25.361903   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:15:35.368Z] E0617 00:15:35.367817   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
E0617 00:15:35.367899   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:15:35.367817   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\nE0617 00:15:35.367899   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:15:45.361Z] E0617 00:15:45.361158   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:15:45.361158   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:15:55.358Z] E0617 00:15:55.358669   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:15:55.358669   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:16:05.362Z] E0617 00:16:05.362089   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:16:05.362089   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:16:15.373Z] E0617 00:16:15.372940   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:16:15.372940   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:16:25.366Z] E0617 00:16:25.366753   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:16:25.366753   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:16:35.374Z] E0617 00:16:35.374273   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:16:35.374273   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:16:45.375Z] E0617 00:16:45.374748   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:16:45.374748   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:16:55.375Z] E0617 00:16:55.374861   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:16:55.374861   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:17:05.365Z] E0617 00:17:05.364565   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:17:05.364565   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:17:15.372Z] E0617 00:17:15.372025   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:17:15.372025   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:17:25.374Z] E0617 00:17:25.373538   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:17:25.373538   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:17:35.366Z] E0617 00:17:35.365922   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:17:35.365922   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:17:35.366Z] E0617 00:17:35.365993   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:17:35.365993   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:17:45.372Z] E0617 00:17:45.372466   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:17:45.372466   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:17:55.370Z] E0617 00:17:55.370151   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:17:55.370151   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:18:05.366Z] E0617 00:18:05.366464   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:18:05.366464   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:18:15.366Z] E0617 00:18:15.366362   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:18:15.366362   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:18:25.367Z] E0617 00:18:25.366990   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:18:25.366990   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:18:35.367Z] E0617 00:18:35.367082   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:18:35.367082   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:18:45.365Z] E0617 00:18:45.364930   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:18:45.364930   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:18:55.360Z] E0617 00:18:55.360604   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:18:55.360604   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:19:05.360Z] E0617 00:19:05.360469   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:19:05.360469   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:19:15.368Z] E0617 00:19:15.368280   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:19:15.368280   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:19:25.363Z] E0617 00:19:25.363489   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:19:25.363489   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:19:35.371Z] E0617 00:19:35.370710   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:19:35.370710   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:19:35.372Z] E0617 00:19:35.371302   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:19:35.371302   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:19:45.377Z] E0617 00:19:45.372757   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:19:45.372757   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:19:55.366Z] E0617 00:19:55.365889   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:19:55.365889   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:20:05.370Z] E0617 00:20:05.370239   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:20:05.370239   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:20:15.373Z] E0617 00:20:15.373083   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:20:15.373083   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:20:25.359Z] E0617 00:20:25.358638   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:20:25.358638   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:20:35.359Z] E0617 00:20:35.358746   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:20:35.358746   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:20:45.360Z] E0617 00:20:45.361146   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:20:45.361146   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:20:55.375Z] E0617 00:20:55.375434   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:20:55.375434   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:21:05.361Z] E0617 00:21:05.362396   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:21:05.362396   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:21:15.365Z] E0617 00:21:15.365808   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:21:15.365808   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:21:25.363Z] E0617 00:21:25.364169   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:21:25.364169   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:21:35.363Z] E0617 00:21:35.364060   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:21:35.364060   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:21:35.364Z] E0617 00:21:35.364060   10992 watch.go:76] watchAllDir unable to walk dir `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`: on path "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect" (DirEntry: <nil>): CreateFile c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:21:35.364060   10992 watch.go:76] watchAllDir unable to walk dir `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`: on path \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\" (DirEntry: <nil>): CreateFile c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-06-17T07:21:45.368Z] E0617 00:21:45.368453   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:21:45.368453   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:21:55.375Z] E0617 00:21:55.374709   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:21:55.374709   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:22:05.369Z] E0617 00:22:05.369335   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:22:05.369335   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:22:15.367Z] E0617 00:22:15.367547   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:22:15.367547   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:22:25.362Z] E0617 00:22:25.361943   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:22:25.361943   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:22:35.372Z] E0617 00:22:35.372423   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:22:35.372423   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
[debug] [2025-06-17T07:22:45.355Z] E0617 00:22:45.360731   10992 load.go:23] Could not load config from "c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect": could not find dataconnect.yaml in `c:\Users\cmill\Projects\FiberFriends\FiberFriends\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0617 00:22:45.360731   10992 load.go:23] Could not load config from \"c:\\\\Users\\\\cmill\\\\Projects\\\\FiberFriends\\\\FiberFriends\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\cmill\\Projects\\FiberFriends\\FiberFriends\\dataconnect`\n"}}
